content/en/docs/collector/troubleshooting.md

```diff
@@ -15,6 +15,172 @@ network issues, it can be helpful to send a small amount of data to a collector
 configured to output to local logs. For details, see
 [Local exporters](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/troubleshooting.md#local-exporters).
 
+## Check available components in the collector
```

Comment:
```suggestion
## Check available components in the Collector
```

Reply:
@theletterf do I remember correctly that we agreed that we write "Collector" if we are referring to the OpenTelemetry Collector?